oem/ibm/libpldmresponder/file_io_type_lid.hpp

```diff
@@ -278,12 +304,39 @@ class LidHandler : public FileHandler
                     oemIbmPlatformHandler = dynamic_cast<
                         pldm::responder::oem_ibm_platform::Handler*>(
                         oemPlatformHandler);
+                // validate api
+                auto& bus = pldm::utils::DBusHandler::getBus();
```

Comment:
scope of this can be reduced , can be moved to line number 315

Reply:
done